packages/app/src/components/Page/TagsInput.tsx

```diff
@@ -38,11 +38,10 @@ const TagsInput: FC<Props> = (props: Props) => {
   const searchHandler = useCallback(async(query: string) => {
     const tagsSearchData = tagsSearch?.tags || [];
     setSearchQuery(query);
-
-    tagsSearchData.unshift(searchQuery);
+    tagsSearchData.unshift(query);
     setResultTags(Array.from(new Set(tagsSearchData)));
 
-  }, [searchQuery, tagsSearch?.tags]);
+  }, [tagsSearch?.tags]);
```

Comment:
This fix should be good 👍 👍 👍 👍 👍  cc @yuki-takei @Yohei-Shiina 

↓ the cause of the bug ↓ it's totally my fault 😭🙏 
https://github.com/weseek/growi/pull/6062/files#r909430348

## Issue
https://github.com/weseek/growi/issues/6169